api/src/dtos/fantasy-teams.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { IsString, IsNotEmpty, MinLength, MaxLength, IsNumber } from 'class-validator';
+
+export class CreateFantasyTeamDto {
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(1)
+  @MaxLength(40)
+  public team_name: string;
+
+  @IsNumber()
+  public owner: number;
+
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(1)
+  public logo: string;
+
+  public points: number;
+
+  public qb: number;
+  public rb1: number;
+  public rb2: number;
+  public wr1: number;
+  public wr2: number;
+  public te: number;
+  public pk: number;
```

Comment:
Yeah I realized that our external api uses PK and so it was causing issues when checking positions. We should still display K though on the front end because it makes more sense

Reply:
Ah ok, that's a good catch